eng/pipelines/templates/jobs/ci.yml

```diff
@@ -233,10 +235,22 @@ jobs:
       MatrixFilters: ${{ parameters.MatrixFilters }}
```

Comment:
You are supposed to include the archetype-sdk-tests-generate.yml twice. Once for the normal test generation and then once for the dependency test generation.